### PR TITLE
Use laser odom

### PR DESCRIPTION
**Pull request by 534o**
gazebo: I'm not sure but laser odom is not work on gazebo..
realrobot : something wrong on odometory

### CODE REVIEW

File: spur_gazebo/launch/spur_world.launch [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <arg name="headless" default="false"/>
   <arg name="debug" default="false"/>
   <arg name="sec_idle" default="1.0" />
+  <arg name="use_base_odom" default="true" />
```
Comment (130s) [inline spur_gazebo/launch/spur_world.launch]: I'm confused. 

> gazebo: I'm not sure but laser odom is not work on gazebo..

So did you mean `use_base_odom` should be false here??

**Comment by 534o** [comment]
use_base_odom publishes odom from base_controller.py 
https://github.com/534o/spur/commit/189715c3f6bde5e34aed923f16d361829413efce
https://github.com/534o/spur/commit/0db282dc9dce876cc6a0b756e8f1e960d0721b1a#diff-9fd72d63530e09f097329895712187d1R8

and when you use 'laser odom", we do not publish odom from base_controller, but push odom from https://github.com/534o/spur/commit/0db282dc9dce876cc6a0b756e8f1e960d0721b1a#diff-9fd72d63530e09f097329895712187d1R8
laser_scan_matcher

on gazeob, laser_scan_matcher does not work well but odom from base_controller.py seems ok.

**Comment by 130s** [comment]
So the source of odom <nav_msgs/Odometry> is:

- Sim: base_controller
- Real: laser_scan_matcher? Looks like it only publishes [geometry_msgs/Pose2D](http://wiki.ros.org/laser_scan_matcher#Published_Topics)

(And by `laser odom` I assume you mean `robot_pose_ekf` in this context.)

**Comment by 534o** [comment]
Both robot_pose_ekf and laser_scan_matcher publishes tf from odom->base_footprint and one takes /odom to calculate the transformation and the other uses /scan.

**Comment by 534o** [comment]
Please check #52 we use base odom for both sim and real so you do not have to confuse on this topic

**Pull request merged**